code/crates/core-consensus/src/effect.rs

```diff
@@ -84,6 +84,11 @@ where
     /// Resume with: [`resume::Continue`]
     Publish(SignedConsensusMsg<Ctx>, resume::Continue),
```

Comment:
Should we consider renaming `Publish` to `PublishConsensusMsg`?

Reply:
2526b2c017db83ed680f800c59a6efea783246d1